packages/tools/src/utilities/livewire/LivewireScissors.ts

```diff
@@ -102,6 +104,48 @@ export class LivewireScissors {
     this.priorityQueueNew.push(startPointIndex);
   }
 
+  /**
+   * Returns a smoothing path count for how many items to remove.
+   * This will remove points, up to count which have a high gradient, up to
+   * count of them where the clip value is larger than that provided.
+   *
+   * @returns Count of items to remove from the path.
+   */
+  public smoothPathCount(
+    pathPoints: Types.Point2[],
+    lastPoint: Types.Point2,
+    count = 5,
+    clipValue = 0.85
+  ) {
+    const lastIndex =
+      (lastPoint &&
+        pathPoints.findIndex((point) => isEqual(point, lastPoint))) ||
+      -1;
+    if (pathPoints.length - lastIndex < count * 2) {
+      // If a nearby point is clicked, just add it anyways, because that means
+      // the user actually wants the given point
+      return 0;
+    }
+    let removeCount = 0;
+    for (
+      let i = pathPoints.length - 1;
+      i > pathPoints.length - count && i > 0;
+      i--
+    ) {
+      const weighted = this._getWeightedDistance(
+        pathPoints[i],
+        pathPoints[i - 1]
+      );
```

Comment:
I thought this function should be in `LivewirePath` but there is a dependency to `_getWeightedDistance` that does not allow it to be move :disappointed:

Reply:
It specifically requires the livewire scissors data.  There are going to be other additions to livewire scissors to deal with the interpolation changes.